src/NServiceBus.Monitoring/ProcessingTime/ReportProcessingTime.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace NServiceBus.Monitoring.ProcessingTime
+{
+    using System;
+    using System.Diagnostics;
+
+    class ReportProcessingTime
+    {
+        public void Report(DateTime timeSent, string processedMessageType,  DateTime startedAt, DateTime completedAt)
+        {
+            var computedTimeElapsed = ProcessingTimeCalculator.Calculate(startedAt, completedAt);
+
+            var totalTimeInSeconds = Convert.ToInt32(computedTimeElapsed);
```

Comment:
Should this time be measured in seconds or milliseconds?

`TimeSpan` has a `TotalSeconds` (and a `TotalMilliseconds`) property on it.